models/discord-guild.js

```diff
@@ -13,8 +13,10 @@ let guild = new Schema({
     frontDeskChannelID: String,
     logChannelID: String,
     companionChannelID: String,
-    ignoreUsers: [String],
+    ignoreUsers: [String],  // Todo: No idea what these 2 fields are getting used for
```

Comment:
I'm pretty sure these were added with the view that they will be used for this type of feature, but it was never implemented. Can you check they're not used anywhere and then use or delete them to avoid confusion later?

Reply:
I made a cursory search (all files with ignoreUsers) but didn't find any reference. Didn't remove since it was added with the very first commit of the file. I will check once more and remove if necessary.